drivers/ukvirtio/mmio/Makefile.uk

```diff
@@ -0,0 +1,12 @@
+$(eval $(call addplatlib_s,kvm,libukvirtio_mmio,$(CONFIG_LIBUKVIRTIO_MMIO)))
```

Comment:
Same comment apply here as in `virtio_9p`, `virtio_net`, etc.

You also need to create a `Config.uk` for the MMIO transport:
```
+config VIRTIO_MMIO
+       bool "Virtio MMIO transport"
+       default n
+       select VIRTIO_BUS
+       select LIBUKOFW
+       select LIBUKINTCTLR_GIC
+       depends on ARCH_ARM_64
+       help
+              Virtio MMIO transport.
```